#request
geometry_msgs/Pose T_0_FL
float64[] trocarAngles
---
#response
geometry_msgs/Pose trocar

